.github/workflows/build-and-test.yaml

```diff
@@ -85,33 +94,12 @@ jobs:
         run: |
           cd tests/third_party/yosys
           sudo make install
-      - name: Find out Slang revision
-        run: |
-          {
-            echo -n "SLANG_HASH="
-            git submodule status --cached third_party/slang | awk '{print $1}'
-          } >> $GITHUB_ENV
-      - name: Find out Makefile revision
-        run: |
-          {
-            echo -n "MAKEFILE_HASH="
-            git log -n 1 --pretty=format:%H -- Makefile
-          } >> $GITHUB_ENV
-      - name: Cache Slang build
-        id: cache-slang
-        uses: actions/cache@v4
-        env:
-          cache-name: cache-slang
-        with:
-          path: build/slang_install
-          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ env.SLANG_HASH }}-${{ env.MAKEFILE_HASH }}
-      - if: ${{ steps.cache-slang.outputs.cache-hit != 'true' }}
-        name: Pull Slang
+      - name: Pull Slang
         run: |
           git submodule update --depth 0 -- third_party/slang
```

Comment:
Don't we need fmt here?

Reply:
I don't generally update utility dependencies like this since it mostly just results in spurious rebuilds and no actual benefit.

Reply:
Where is fmt coming from if we haven't downloaded the submodule?

Reply:
We have; see `submodules: recursive` earlier. If you somehow run `cmake` without the submodule the build fails.

Reply:
Oh wait, I completely misunderstood what this step does because I read the name and not the contents. You are not *pulling* (merging with remote branch) Slang, you are *fetching and checking out* clang.

Reply:
I see, we weren't using `submodules: recursive` for historical reasons (there was a large submodule which we wanted to ignore in CI)